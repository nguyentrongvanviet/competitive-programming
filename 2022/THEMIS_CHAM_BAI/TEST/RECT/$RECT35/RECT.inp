967
366784055 590258109
797361168 150929142
-614099437 -42484943
-15907430 -965760525
-174145508 -667177666
535992077 -721839553
-695845821 305184914
426309951 480161152
235323461 333906703
907249758 506551160
397380514 -930196116
-531051098 -3120891
-480018327 599758893
-912270986 713070290
107124703 847940618
-547627171 596210088
-109334303 -660304791
-900818797 -224192700
-109577931 -821767325
-545106401 670602289
-938563832 -965812674
-290430155 -390729239
-866152211 -951975432
-556491976 -983033321
509271220 194007117
-472290044 -553268238
175499987 88107131
-526049518 949898
833138357 821500849
-878155044 865458688
520150422 265335853
-381002651 -235244306
-211848388 -398484133
-369912456 -203695755
-900302164 931567742
-15907430 -965760525
-825737496 -89421864
455175800 -840363840
-290767307 996069860
-608631643 -550550140
233856941 -367307430
-3411977 328999516
546926045 814602657
770357133 490654434
-323883197 -779463900
-287665037 -522766722
-881568903 -540427212
-924479739 -664331644
-841750248 -59142038
-295229443 -74827281
483236941 -445294329
-458145582 -859884898
903602696 -950835409
-420008951 -103057464
-717954599 961906845
15157928 631927560
160984707 -630478664
426309951 480161152
27403718 753728179
-868423262 691719042
-872928769 -778357168
237594066 302109313
284141652 -668472071
891918935 932035130
232637043 339551160
-787524952 925071310
-346228632 -970233926
-386909315 212564392
134988699 -947760666
-138495613 458243561
140008081 846878172
140008081 846878172
316333797 -140267837
-239929321 124618593
436145881 926982724
682379856 -216083818
-774060312 441262391
-809558317 -883267396
-195919276 -713864600
331450042 -232105380
-559492583 200170841
321053435 -367580486
-519529210 308992570
-662394911 -465050994
-383354236 -490144979
-80445772 -728915456
-25736954 319239427
445224008 337221637
306939706 413807867
-157151319 -493603853
374139344 231873011
-292155204 -657960370
916775615 527726316
556293033 318781491
514882935 299473360
-127278240 -190554194
-160741931 572009104
483236941 -445294329
-348647146 -150921713
-143736266 909645626
371190718 843237924
-523226936 -568643632
252786054 150227553
326837137 -845448147
-886356098 774937735
968494566 -26113882
182351430 490500617
-317778443 -297468062
461709316 -532289981
351366399 957483810
-156701911 -708712967
-38279081 369992434
351607917 448248304
608044034 -729086078
342987992 -60761419
-517560245 60692202
-535990001 -735334130
-718853905 -726107267
152993948 729726282
-334277108 492833484
241535051 207276425
-670937661 420908255
-743734208 -164392385
529063124 -106227076
765185386 633756551
-76942990 -262458199
704799677 996952207
302888855 26793111
391657375 -634932584
760311866 -913162750
960391263 291831040
-129391201 830009763
-898767436 239340641
-515890298 -691123222
-504955924 -481945975
152993948 729726282
-83131152 -104882043
302943877 32188695
405497693 380686606
-650281059 884702744
-723808360 -222373414
726298992 -551678757
333101880 735658315
-294103050 539884858
-24207707 -95058931
-252365259 392878933
519488674 555770115
-269203715 204990197
770775315 507051448
864353514 -360563794
107004954 159949105
-752827336 380502318
-475085866 369861913
-493710081 -483568216
-807609969 -859518770
593651257 -256515222
-622773817 322186359
-270137439 794900189
859131324 866165052
61099532 -874194851
127265285 -512393581
-997571309 -372680945
-196299094 56748313
-506454231 -382560179
208672874 185769162
-620469844 -750194445
436938616 999801445
-38452242 -592138066
-778991463 898679368
-209435297 -962330568
92128099 274494837
-155127824 -181652969
440022565 183356763
-22788818 689235006
847846306 710596026
-934781429 803418497
565226907 -63970213
23561629 -750039831
-550726626 -66324550
-636596840 543967976
-194951206 -370653367
-127545959 -744513891
-946173620 98945374
-995786713 -119450009
-236700635 346178785
-784249320 -49250486
92092978 523771307
842306308 -720044935
66055223 -912901744
-773294691 -688121793
191628935 376790428
-574703256 543574106
611155845 -590662019
-323883197 -779463900
419916882 -696333697
-714485728 -161245636
-610923897 -241142465
500061888 934846886
751720100 -710911896
-640444813 -883012677
-804745699 -390535424
385206807 788085419
-542949749 -166217950
-238471932 52848975
957240031 -163636423
343743027 -827896645
336970871 -779251628
-768793894 -902298674
-41657856 -79126247
897883507 -960619269
-437985243 936820955
263916129 67953124
357020955 619709159
-34956526 506270463
536647459 -981167442
-873730940 -309556394
842576262 -377112716
-809558317 -883267396
842576262 -377112716
-842938848 -335869315
330473949 645632780
606053054 577696921
-622591382 -597012522
595659097 615909601
-446358119 235075693
-912836576 -661844880
465521633 -812930355
-654618577 58163035
-344277712 928261545
-828203361 -18903578
-436912154 -625714694
391688515 -59521071
-575620038 41949514
-206643494 -905739676
677040496 767865124
-587710369 -996111505
204893323 313769507
946361997 -341064634
-746507010 -838033716
-129564717 72476577
602447845 -405317623
-778227717 -954119763
-146859318 -730882992
150441743 306309395
-640236276 21706681
-447463030 364424403
-466821856 -532686026
-88509559 218998104
-526711804 392752475
221256689 -618892725
-275118305 -355972142
-85872544 530435171
533804122 54061358
-725729922 775604782
-680539273 -840360281
-249050877 -780889562
-590809277 -436271021
404757541 -90555155
152176528 652469619
581309078 766556031
-140651538 -734889099
72325989 -160153970
-453811370 487560191
-34942826 -269049500
-404650170 342470735
51426126 716624077
-803911019 -444073778
628027221 191050624
-691942690 -211742299
-102556162 607715333
-798174236 -260500418
-170009887 112268953
-72744031 216281214
735791916 662697514
-913567506 -779368420
526409242 486464876
701336124 705275395
848314629 -557290952
222503912 380764414
-391914545 -454694190
307763830 -625552993
-694465105 -297462390
-213036293 -184329489
593482574 776953144
-394466943 359012512
-718782265 -322445516
128257175 -46750429
-230103868 -516809355
371190718 843237924
-211848388 -398484133
842952134 116755594
-856803277 -267667890
685749492 416413777
-409682248 509092310
-129391201 830009763
379579197 -840171743
-412844425 -236280916
671892736 716470413
-837758652 -693095134
-935231711 -340533829
646319505 113876952
-209435297 -962330568
-230103868 -516809355
878162204 138671071
-872928769 -778357168
795209166 -159516783
367142026 708107776
-801138583 964788422
315758066 55067069
-517986285 149798582
-795871219 -63143677
-69216088 -772628616
425632542 289015209
-590681771 641591943
417822552 431811160
185275357 -216981468
60989607 -164382384
-97954898 504977162
61099532 -874194851
412907175 225745847
289730689 674935761
-165186109 -166870242
-446635770 -624817996
373477100 -577789765
965564694 -497005663
-742285220 -545082364
-545106401 670602289
786523527 -600292855
-166942320 -392061986
842952134 116755594
-532783602 622902278
316161477 -66668061
-449397414 -86800460
284141652 -668472071
500061888 934846886
-825902374 495959029
-65294836 -824571936
637242131 402727812
857557359 -349881177
-617293114 -347458515
847960109 -810959913
395049233 291177591
-531704400 257454761
321053435 -367580486
-354827187 264677053
388010867 110301293
271783269 412716240
-260567290 153057095
580815820 812708548
-496217927 334372576
307172541 -849668643
646172385 -413994532
-839274896 -453000989
509205747 -913228392
956874365 -469982045
-566004550 639767715
419916882 -696333697
715541002 834731835
208672874 185769162
-16043661 606956615
313264815 959311421
-954540834 -170134934
700697397 81829107
-531675871 215937610
-426922075 -847775026
-806947480 197027584
690631705 56742085
441290166 -490489103
-665817263 779170833
152145686 617370987
-88191319 64358687
361558460 -255455867
709500485 -415747415
642676855 -581420840
-110443624 346566528
179626823 799967622
9997944 -630875361
-581701367 -251461196
-64739553 -678919494
-697933387 -222059392
17946254 519177576
-652325317 583497142
-22788818 689235006
-490747920 404898683
-87723852 -478814297
-926578677 -261050739
-39142817 262993628
-539081457 -466477585
268113957 -239065991
687326542 862762838
-204469738 -419429284
26044882 -84507616
-609679006 -589393729
127462383 848167745
-732345509 -278293510
216588730 -504865014
102840925 850346566
-776112909 947746675
-700440039 510923708
525730081 -381545013
-87723852 -478814297
187068107 -876124532
755338513 963877431
893504914 344741490
365396096 -898853675
56601972 522656661
-650281059 884702744
-240023128 -471874847
-301476841 -341551138
242724840 612384791
-309462016 995966110
-47038880 480812454
-409394330 377644579
98699628 -115010078
-895758992 -123146235
-336525865 -294752892
15838808 887210249
-702148294 -218466206
751720100 -710911896
-99488009 17579439
580681483 76852437
-344989802 979975318
61741727 -562410964
-769017694 -43656472
-938050121 337573667
-76208965 -234300665
515232082 -320784328
-104739292 421190199
-173646999 -580971069
-387774401 451144664
646364127 -921439624
596562145 -457980795
188555113 655128145
399686824 -173408135
646172385 -413994532
811845662 -820545061
-433018377 -584308457
-590809277 -436271021
924927798 200207615
-887851890 -916386445
550453811 431043570
-825737496 -89421864
-375109885 -54909851
893504914 344741490
811845662 -820545061
-348647146 -150921713
316276430 884087833
34100787 -713239114
-977975328 239360133
-914030487 -494898726
-868423262 691719042
-560674507 -383368918
-784249320 -49250486
-998217798 291853253
-652325317 583497142
-721997377 -229402006
388010867 110301293
-949084649 392648634
317900798 -510899706
-970406560 -422548196
478161920 -23773583
-129747607 -995228751
-605803003 -426233437
22897822 -508658158
374773662 -184060071
-167642829 -683286400
684836550 -979737645
-339068435 -126984038
874897080 725384677
879320841 478487837
-243553239 490352124
-938050121 337573667
-869297718 -949916621
987938873 -350933788
-232610502 -478393015
593651257 -256515222
-760914980 701314680
245172472 -170981837
-152972798 265475486
591394396 678391970
447874169 188113811
257155426 -807568919
-551332199 -296605487
464890775 -765736157
780881806 255107377
52571231 65215176
-844075596 -892570145
-597009011 44439032
-313682670 591594143
-850977681 752443677
316333797 -140267837
314031676 418807833
-390525027 -468198204
646319505 113876952
-547812900 -320444465
185646752 553498996
549770384 -99529891
-397922679 -489584447
-112457309 -40425494
656127362 849045898
-732345509 -278293510
60989607 -164382384
-725594873 751289085
950837531 -769895046
-851144883 -624748209
846157605 535192453
501538288 196920700
-487909831 -909898237
-992576647 -738264651
95780345 207282021
-946173620 98945374
-5854988 598021558
-472290044 -553268238
734000868 545000455
-836184272 658586070
-232610502 -478393015
775435820 -304022237
328008913 789652614
397967933 150106451
-552954834 -159621580
13138178 87918819
-882567366 -146365654
-58342495 -678465069
640940830 450953007
374070143 -381302805
-783275488 535883639
-332139397 -457293746
-494750767 660887560
-9573238 19088287
171712856 83464942
-458145582 -859884898
394809244 -491085839
452948248 546833632
274212349 -856829296
168042836 229652318
-80281416 259090645
-734891167 -73386347
339232060 -738510485
444662569 743785892
342049409 392144492
444662569 743785892
242724840 612384791
127265285 -512393581
-631215672 694849522
-72278024 -956708146
-785299759 -318817119
-679949833 710111128
215684507 55920295
-917313036 -732684052
-689593893 626083241
942249043 951849240
-217800136 567182756
-885091345 501971044
283315773 -653949837
-547812900 -320444465
-344989802 979975318
-808165437 869536744
746596721 291052012
451053441 -790314962
-869297718 -949916621
-364420389 273762951
-700593846 516369011
-170009887 112268953
102840925 850346566
236328499 -666411371
-919834983 -306915997
768189609 633512437
-198716530 -723983739
747930436 996022791
-648029728 -719468621
153260521 768313994
110869685 -275894297
-532307700 -79284521
-960516897 552759158
-742285220 -545082364
-523669470 501852536
-743314923 -177140161
-766706985 -400783575
-777329240 64703537
15838808 887210249
-804745699 -390535424
-120093412 517813511
703582825 -712551203
307046970 -124363406
139304738 397151707
738162949 843992997
-502713589 330491314
-764919672 -312608123
281963590 -95921339
-365212971 145437167
950871693 -185084933
-384433471 153699393
924531762 -273571985
-555799539 -459588963
-152898423 80939627
-865633994 -269272660
909554246 -949610430
-31537958 910017350
786905797 -324087971
-58342495 -678465069
-960516897 552759158
-154314376 450030113
57859576 502135620
25259477 516386044
362710250 181525296
-689879081 759244215
190066029 116401360
-721997377 -229402006
573074188 -948925524
-285330742 -103265956
718237387 -342298733
630963105 415570746
-930051245 20779758
-812298279 237962293
157474339 535562585
-304564051 312107909
-968439121 778082371
339267714 72802169
-700629742 -586926531
326577108 -42038196
37783360 236800959
724655261 -177209207
-349589553 -365153158
773367802 550965984
-129747607 -995228751
-896762796 -196108531
362703157 -43238759
265060395 378282542
-915609449 -78432380
-437985243 936820955
-510795937 457906715
274212349 -856829296
-627832805 720071576
160984707 -630478664
821209164 -351363462
711049341 555789590
549688127 600531300
-110443624 346566528
192114205 884451701
903656233 53396046
-835078538 603968156
388273889 -465646609
985058117 -782107344
-298280523 872767117
95780345 207282021
397380514 -930196116
-750632917 594142376
-504955924 -481945975
-752488821 619373965
-771040315 -101491190
-828055279 -990592834
509205747 -913228392
334295345 -242351008
858350616 -287390342
92920526 199990797
-502713589 330491314
328771932 -246504569
-710022907 -414279638
932345237 -245464815
834988445 448619524
853518288 -427261148
942249043 951849240
37783360 236800959
-89474077 -378975000
642676855 -581420840
-986977914 -116820468
-485040606 -530504809
489799966 -87201376
-896762796 -196108531
-301750948 -925957607
160742748 -572966565
-851144883 -624748209
231169870 -537501848
73714252 551435586
-361334264 -75343602
-853463034 -151974098
-526049518 949898
390209548 -511608558
493890119 741104458
455175800 -840363840
581280501 -548693141
-660599749 -659317396
-183292459 -73355011
647509616 -334768124
828472429 76097575
-485040606 -530504809
-87337830 533787506
727221840 -340699575
-194377941 -561944451
-839274896 -453000989
323058310 -327414657
-831483348 291928629
423182755 -423206706
690631705 56742085
130214898 -714164688
27507432 185874731
-236095214 -202868343
-104922402 -429383837
395835402 457780755
796142488 675385985
-510701061 6178164
151180844 -251084044
991223347 -810197371
-326107058 -16949427
395948976 -101642546
-267945364 -27014298
485413886 -251550653
-852653168 549075698
-364420389 273762951
572701083 675414789
-901770721 692384896
-397149698 -56372719
-725729922 775604782
387940511 -42437852
268103100 784234922
466533504 28587868
-126644126 -794431089
550236192 -652559039
-809689494 -320644401
-942889047 -884183440
-571085516 990603467
197189336 679784893
371652716 -981163741
985000148 979529408
198307346 -201849964
-841043786 -859663528
-566004550 639767715
909607455 -171824774
-700831167 949356782
732879165 -474962498
-523226936 -568643632
573074188 -948925524
221167766 772318925
-881100117 -733349780
461709316 -532289981
333404820 895167528
446237676 -523382180
-244376657 349443416
42204929 529634454
-219828035 584789949
139274761 -120490569
216553759 20171825
-928588813 -391133493
493520683 -474232759
76504216 -303237743
696941957 -59984131
607101187 526076561
174273171 616242705
524922904 -158717750
-962389017 -907585128
623351291 -826239110
-372320278 331664673
-892868322 -499425268
-494580461 -839452619
692045124 -346548475
-5089304 428533828
-503661673 -968636598
-968344960 -351346821
807370976 -104017906
216588730 -504865014
-149416861 785655719
573613186 -420645721
-468630813 567724300
-711905949 406294704
-351528767 -287808752
-694465105 -297462390
-688729702 178543865
-260567290 153057095
-700593846 516369011
478370614 721526781
-490903331 -318319695
414308292 -991838997
-295229443 -74827281
-932520502 970534995
419260557 924368141
859131324 866165052
-610923897 -241142465
595955151 681258957
729309640 -355501263
965564694 -497005663
-457544194 -386775647
-16043661 606956615
445675560 -35717829
696764612 -5858482
987385961 -173609301
-369765476 956755578
-853716073 -846551442
10274592 -374877576
47169491 889008109
-238062905 165145057
-290767307 996069860
367142026 708107776
110869685 -275894297
-152898423 80939627
775435820 -304022237
846157605 535192453
218606648 -5281171
503392547 860712078
118779745 -405354042
-240524369 538376932
-249050877 -780889562
315758066 55067069
306321974 -229890326
244186746 -231041663
-540864986 428867693
-670465512 808902135
673415473 731148891
151180844 -251084044
307763830 -625552993
41026662 124684682
240533783 -155222571
808121051 -809083504
194632410 -427872954
-45331566 772226385
332658090 551817168
407963122 865574372
427086598 334344689
701336124 705275395
-332411157 -306643592
69859152 -364613869
-703512010 -546928165
-406845921 104597479
159158637 900167371
-354871398 815767967
506496819 -923311484
226164157 523821925
-703477364 295315182
656127362 849045898
427360324 -669083056
160330544 11266967
-337191196 -523640602
316276430 884087833
-173134384 -970308766
-387953311 447536199
989586518 572691813
-725558800 -55211296
89460837 -168506918
-204469738 -419429284
-418175698 463424105
206360601 -424993148
-286712451 935348525
-584401982 595627989
436938616 999801445
-298280523 872767117
506496819 -923311484
995215009 -911890504
-369765476 956755578
-602734788 -451223748
445645719 253739083
429566751 559090165
-466821856 -532686026
648558132 -851600458
176994190 -712823664
127778441 658528513
-952856498 608279709
-221975014 832769192
446705332 587000045
-344598361 -690180862
-287583266 599693855
-745359413 630920620
-183268170 -835199214
801007450 564094177
269674642 197615276
-5534864 -447682479
172902226 -89048815
-921220984 -187085300
-917798658 -367384714
940419489 819093453
-486447920 897524254
77672918 396380099
-9660457 113945203
-578266755 801658933
26044882 -84507616
-152850459 -445133215
-803911019 -444073778
704799677 996952207
-801823409 317390779
119077289 432874598
-873730940 -309556394
-532307700 -79284521
-67438961 217472254
297485408 347422923
-497981634 -673029256
292725896 136624805
969707462 -783538229
-343015730 729132070
-683141341 -169043485
-809689494 -320644401
-952856498 608279709
-661363085 -850782933
-523658943 -780885865
385042376 -108077815
139399825 -731455244
-229914235 351890491
188555113 655128145
-841750248 -59142038
-681553825 -490598836
471732169 334322287
47654632 581693858
323152484 -619734313
-126437309 658413943
-955741391 895466715
204893323 313769507
628027221 191050624
-350485679 866521469
-742821800 -8901982
-365042277 255866860
903602696 -950835409
-212585102 -869571832
933196817 -946799252
-451805723 641297929
-168935250 454062940
618725456 636267873
502667509 757503515
156903916 -420188272
-587710369 -996111505
-334384485 -422911489
-835078538 603968156
92018599 -218984032
740906200 805179452
912488380 -401383917
314031676 418807833
-390525027 -468198204
-927046042 291032611
-394466943 359012512
-828055279 -990592834
130754052 -732479865
743032204 -283559629
159158637 900167371
-356367976 706647574
7858015 403788737
67222024 234227131
668117207 -278434894
365396096 -898853675
-17099064 -67801916
423368081 430980853
-714485728 -161245636
-634676240 -759346917
709500485 -415747415
139274761 -120490569
161063347 773531650
-336525865 -294752892
905314469 197340574
-542949749 -166217950
157474339 535562585
182951206 618258631
-968439121 778082371
859645556 -348366065
-532325316 851604348
298432538 -878843972
-706400013 -177570948
-953231702 138237362
-166550544 742852001
565732432 -367702428
-523658943 -780885865
797361168 150929142
-293164312 -324786666
932345237 -245464815
-744594675 -207209092
66055223 -912901744
-354173077 -214213025
801228207 -890534095
615342346 133329629
-970406560 -422548196
466533504 28587868
-620948775 470532004
31285757 -996519816
