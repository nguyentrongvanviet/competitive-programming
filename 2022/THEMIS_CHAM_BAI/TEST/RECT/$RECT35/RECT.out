751425631211667990